bundles/core/pom.xml

```diff
@@ -150,6 +150,7 @@
                                         <!-- private enum; cannot test valueOf / values -->
                                         <exclude>**/SocialMediaHelperImpl$WebsiteMetadata$Type.class</exclude>
                                         <exclude>**/AdaptiveImageServlet$Source.class</exclude>
+                                        <exclude>**/HierarchyComponentContextWrapper.class</exclude>
```

Comment:
Please don't exclude this from coverage checks!

Reply:
can you please help me with your best practices for testing a POJO? 

Should it be just tested as a by-product of the Integrated Tests or should it have a separate file?

When, before creating this PR, I attempted to create an UnitTest for this class it was just containing mocks and checks if the mocked values were returned, like:
```
private ComponentContext getMockedComponentContext() {
    ComponentContext componentContext = mock(ComponentContext.class);
    when(componentContext.isRoot()).thenReturn(Boolean.TRUE);
    ...
    return componentContext;
}

@Test
public void test() {
    ComponentContext componentContext = getMockedComponentContext();
    HierarchyComponentContextWrapper hccw = new HierarchyComponentContextWrapper(componentContext, ...);
    ...
    assertEquals(componentContext.isRoot(), hccw.isRoot());
}
```
so I thought that it does not bring any value and dropped the idea.

Reply:
Generally unit tests and integration tests should overlap as they cover different aspects.

For such a simple wrapper it may seem redundant but having unit tests guarantees the right properties are passed through and only the intended ones are altered.